specification/redisenterprise/resource-manager/readme.csharp.md

```diff
@@ -12,4 +12,15 @@ csharp:
   client-side-validation: false
   namespace: Microsoft.Azure.Management.RedisEnterprise
   output-folder: $(csharp-sdks-folder)/redisenterprise/Microsoft.Azure.Management.RedisEnterprise/src/Generated
+
+directive:
+  # Rename constants in the generated code
+  - from: source-file-csharp
+    where: $
+    transform: >-
+      return $.
+        replace( /public const string Ones /g, "public const string OneSecond " ).
+        replace( /public const string Oneh /g, "public const string OneHour " ).
+        replace( /public const string Sixh /g, "public const string SixHours " ).
+        replace( /public const string OneTwoh /g, "public const string TwelveHours " )
```

Comment:
Do we get default good names for our TLS version string enumeration values? Or are they "onePointOh"?

Reply:
I checked what other services are using, and they seem to be using the same as what RedisEnterprise will use and what Redis currently uses:

`PS C:\repos\feb21\azure-sdk-for-net\sdk> dir -Recurse -Include *.cs | Select-String -Pattern "OneFullStop"

frontdoor\Microsoft.Azure.Management.FrontDoor\src\Generated\Models\MinimumTLSVersion.cs:19:        public const string OneFullStopZero = "1.0";
frontdoor\Microsoft.Azure.Management.FrontDoor\src\Generated\Models\MinimumTLSVersion.cs:20:        public const string OneFullStopTwo = "1.2";
redis\Microsoft.Azure.Management.RedisCache\src\Generated\Models\TlsVersion.cs:19:        public const string OneFullStopZero = "1.0";
redis\Microsoft.Azure.Management.RedisCache\src\Generated\Models\TlsVersion.cs:20:        public const string OneFullStopOne = "1.1";
redis\Microsoft.Azure.Management.RedisCache\src\Generated\Models\TlsVersion.cs:21:        public const string OneFullStopTwo = "1.2";
redis\Microsoft.Azure.Management.RedisCache\tests\InMemoryTests\CreateOrUpdateTests.cs:65:                                                                                MinimumTlsVersion = TlsVersion.OneFullStopTwo
redis\Microsoft.Azure.Management.RedisCache\tests\InMemoryTests\CreateOrUpdateTests.cs:82:            Assert.Equal(TlsVersion.OneFullStopTwo, response.MinimumTlsVersion);
redis\Microsoft.Azure.Management.RedisCache\tests\InMemoryTests\CreateOrUpdateTests.cs:285:                                                                                MinimumTlsVersion = TlsVersion.OneFullStopTwo,
redis\Microsoft.Azure.Management.RedisCache\tests\InMemoryTests\CreateOrUpdateTests.cs:304:            Assert.Equal(TlsVersion.OneFullStopTwo, response.MinimumTlsVersion);
redis\Microsoft.Azure.Management.RedisCache\tests\InMemoryTests\GetTests.cs:68:            Assert.Equal(TlsVersion.OneFullStopTwo, response.MinimumTlsVersion);
redis\Microsoft.Azure.Management.RedisCache\tests\InMemoryTests\ListTests.cs:104:                    Assert.Equal(TlsVersion.OneFullStopTwo, response.MinimumTlsVersion);
redis\Microsoft.Azure.Management.RedisCache\tests\ScenarioTests\BeginCreateFunctionalTests.cs:38:                                            MinimumTlsVersion = TlsVersion.OneFullStopTwo,
redis\Microsoft.Azure.Management.RedisCache\tests\ScenarioTests\BeginCreateFunctionalTests.cs:47:                Assert.Equal(TlsVersion.OneFullStopTwo, response.MinimumTlsVersion);
redisenterprise\Microsoft.Azure.Management.RedisEnterpriseCache\src\Generated\Models\TlsVersion.cs:19:        public const string OneFullStopZero = "1.0";
redisenterprise\Microsoft.Azure.Management.RedisEnterpriseCache\src\Generated\Models\TlsVersion.cs:20:        public const string OneFullStopOne = "1.1";
redisenterprise\Microsoft.Azure.Management.RedisEnterpriseCache\src\Generated\Models\TlsVersion.cs:21:        public const string OneFullStopTwo = "1.2";
redisenterprise\Microsoft.Azure.Management.RedisEnterpriseCache\tests\ScenarioTests\BeginCreateFunctionalTests.cs:41:                                              MinimumTlsVersion = TlsVersion.OneFullStopTwo,
streamanalytics\Microsoft.Azure.Management.StreamAnalytics\src\Generated\Models\CompatibilityLevel.cs:19:        public const string OneFullStopZero = "1.0";
streamanalytics\Microsoft.Azure.Management.StreamAnalytics\tests\ScenarioTests\StreamingJobTests.ScenarioTests.cs:50:                    CompatibilityLevel = CompatibilityLevel.OneFullStopZero,
websites\Microsoft.Azure.Management.WebSites\src\Generated\Models\SupportedTlsVersions.cs:19:        public const string OneFullStopZero = "1.0";
websites\Microsoft.Azure.Management.WebSites\src\Generated\Models\SupportedTlsVersions.cs:20:        public const string OneFullStopOne = "1.1";
websites\Microsoft.Azure.Management.WebSites\src\Generated\Models\SupportedTlsVersions.cs:21:        public const string OneFullStopTwo = "1.2";`

Reply:
which is OneFullStop[Zero|One|Two]

Reply:
we are free to customize it if you wanted something like: OneDot[Zero|One|Two], but then we'd be deviating from what most of the other services are using (aka the default uncustomized consts)

Reply:
The search above is biased, so I ran this to make sure EVERYONE is using the FullStop term instead of Point or Dot: 

dir -Recurse -Include *.cs | Select-String -Pattern "public const string One"

Reply:
British English for the win eh? I am not a fan, I'd think "OneDotOne' is much better, but I suppose we also want to go for consistency with everyone else...